packages/commonwealth/server/controllers/server_communities_methods/update_community.ts

```diff
@@ -6,14 +6,16 @@ import {
   CommunityAttributes,
   UserInstance,
 } from '@hicommonwealth/model';
-import { ChainBase } from '@hicommonwealth/shared';
+import { ChainBase, COMMUNITY_NAME_REGEX } from '@hicommonwealth/shared';
 import { MixpanelCommunityInteractionEvent } from '../../../shared/analytics/types';
 import { urlHasValidHTTPPrefix } from '../../../shared/utils';
 import { ALL_COMMUNITIES } from '../../middleware/databaseValidationService';
 import { TrackOptions } from '../server_analytics_controller';
 import { ServerCommunitiesController } from '../server_communities_controller';
 
 export const Errors = {
+  InvalidCommunityName:
+    'Invalid name, only a-z, A-Z, 0-9, !, @, #, &, (, ), :, _, $, /, \\, |, . and single spaces are allowed',
```

Comment:
the same error text is used in 4 places, maybe it should be the reusable const

Reply:
yes, i couldn't think of where to put it, will add a error messages const maybe